templates/partials/months.html

```diff
@@ -11,7 +11,7 @@
 			{{#ifEquals weight 0}}
 			<div class="n-storylines__heatmap-segment-none"></div>
 			{{else}}
-			<div class="n-storylines__heatmap-segment-colour--background">
+			<div class="n-storylines__heatmap-segment-colour--background all">
```

Comment:
can the class of `all` be better name spaced, eg, `n-storylines__heatmap-segment--all` ?